### DISCUSSION TITLE
Is it possible to use this addon to convert to a VB Project?

### DISCUSSION THREAD

**Discussion by otomay** [joyfullservice/msaccess-vcs-addin#496] [Q&A] [answered]
Hello!

I'm a developer by hobby, never did something in VB or VB.NET, only VBA, so I don't know how to test it.
I did a ms access application for a company and they are considering to not use microsoft office anymore, so my application would not work.

**Comment by joyfullservice** [comment, marked as answer]
To answer your question directly, no, this add-in won't convert a database to a VB Project. (And it is really outside the scope of what this was designed for.) That being said, here are a couple things you might consider looking into.

1. Microsoft Access has a free runtime version that can be used to run a database application even if Microsoft Office is not installed on the computer. This runtime can be freely distributed with your application. (Note that there are a few design considerations to keep in mind when taking this approach.)

2. You might be interested in taking a look at the [TwinBASIC project](https://twinbasic.com/). Coming from a VBA only background, you will probably find that much more familiar and comfortable than taking the leap in to the .NET development paradigm.

**Reply by otomay** [reply]
Much thanks!

I didn't know about the free runtime version!
